src/keycloak/chart/templates/statefulset.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         {{- end }}
        {{- if not .Values.devMode }}
       annotations:
-        postgres-hash: {{ include (print $.Template.BasePath "/secret-postgresql.yaml") . | sha256sum }}        
+        postgres-hash: {{ include (print $.Template.BasePath "/secret-postgresql.yaml") . | sha256sum }}
```

Comment:
The space change here made me think about this - might be good to add a checksum for the realm env secret now since the theme envs could be hot reloaded.

Reply:
I was wondering about how to trigger that earlier. Appreciate the suggestion